code/web/src/modules/StyleSurvey/StyleSurvey.test.js

```diff
@@ -33,6 +34,12 @@ describe('StyleSurvey component', () => {
 			currentIndex: 0,
             survey: 'athletic'
         }
+
+        styleSurveyData.map(surveyData => {
+            return surveyData.images.map(image => {
+                return image.selected = false;
+            })
+        })
```

Comment:
I like that this is working but I want to know if there is another way of doing this just out of curiosity.